src/main/java/org/mule/extension/db/internal/domain/connection/mysql/MySqlConnectionParameters.java

```diff
@@ -115,8 +116,16 @@ public String getUser() {
   private void addMuleLoggerProperty(Map<String, String> connectionProperties) {
     if (connectionProperties != null) {
       try {
-        Thread.currentThread().getContextClassLoader().loadClass(MY_SQL_LOGGER);
```

Comment:
all of this, from the getAvailableInterface() to (most importantly) the generationo of the proxy need to occur using the Extension's classloader.

Reply:
done